lib/fileutils.rb

```diff
@@ -1642,7 +1642,13 @@ def install(src, dest, mode: nil, owner: nil, group: nil, preserve: nil,
       st = File.stat(s)
       unless File.exist?(d) and compare_file(s, d)
         remove_file d, true
-        copy_file s, d
+        if d.end_with?('/')
+          mkdir_p d
+          copy_file s, d + File.basename(s)
+        else
+          mkdir_p File.expand_path('..', d)
+          copy_file s, d
+        end
```

Comment:
I like this.  No dependency modules is great.  What do you think project maintainers?